src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -384,6 +389,14 @@ export default {
 		},
 	},
 	created() {
+		const expCookieSignifier = this.cookieStore.get('kvlendborrowerbeta');
+		if (expCookieSignifier !== 'b') {
+			const { query } = this.$route;
+			const queryString = Object.keys(query)
+				.map(key => `${key}=${query[key]}`)
+				.join('&');
+			this.$router.push(`/lend-classic/${this.$route.params.id}?query=${queryString}`);
```

Comment:
I think there are some other conditions we'll need to check here...and maybe be able to get rid of this exp cookie check altogether...
1. If a loan is `inPfp` we redirect to legacy (https://github.com/kiva/kiva/blob/development/sites/www_kiva/server/views/Xb/Loan/LoanView.php#L95)
2. If the lender is an RTP Translator we skip the experiment (https://github.com/kiva/kiva/blob/development/httpdocs/classes/API/Experiment/LoanDetailBeta.php#L22)

Reply:
@mcstover Sounds good! Is there a way to get if lender has a RTP Volunteer role?

Reply:
@eddieferrer As of my latest merged PR #1 is not true. UI will now render inPfp loans